erdjava/src/test/java/elrond/TransactionTest.java

```diff
@@ -19,12 +19,12 @@ public void shouldSerialize() throws Exception {
         transaction.setGasLimit(50000);
         transaction.setChainID("1");
 
-        String expected = "{\"nonce\":0,\"value\":\"42\",\"receiver\":\"erd1cux02zersde0l7hhklzhywcxk4u9n4py5tdxyx7vrvhnza2r4gmq4vw35r\",\"sender\":\"erd1l453hd0gt5gzdp7czpuall8ggt2dcv5zwmfdf3sd3lguxseux2fsmsgldz\",\"gasPrice\":1000000000,\"gasLimit\":50000,\"chainID\":\"1\",\"version\":1}";
+        String expected = "{'nonce':0,'value':'42','receiver':'erd1cux02zersde0l7hhklzhywcxk4u9n4py5tdxyx7vrvhnza2r4gmq4vw35r','sender':'erd1l453hd0gt5gzdp7czpuall8ggt2dcv5zwmfdf3sd3lguxseux2fsmsgldz','gasPrice':1000000000,'gasLimit':50000,'chainID':'1','version':1}".replace('\'', '"');
         assertEquals(expected, transaction.serialize());
```

Comment:
could be `.replace("'", '"')` but it's not a problem

Reply:
Seems that overload `.replace(String, char)` isn't available (at least not in Java 8).